core/src/com/po/fuck/GeometryMisc.java

```diff
@@ -36,6 +37,18 @@ public static <A extends Entity, B extends Entity> Vector2 direction(A from, B t
         return to.position.cpy().sub(from.position);
     }
 
+    public static <A extends Entity> Entity findClosest(A entity, List<A> list) {
```

Comment:
I'm not sure about typing, should it be `(Entity entity, List<? extends Entity> list)` ?

Reply:
There is already function `public static <A extends C, B extends C, C extends Entity> C closest(A main, ArrayList<B> list)`, use it, but change `ArrayList` into `List`, and I'm not sure about this types with `extends`, maybe rename them into something more meaningful.